templates/404.html.hbs

```diff
@@ -32,13 +32,12 @@
 </head>
 <body>
     {{> includes/header}}
-    <section id="hero">
-        <div class="hero-container">
-            <h1>404</h1>
-            <h2>NOT FOUND</h2>
+    <section id="main404">
+        <div class="main404-container">
+            <h1>This page is not available</h1>
+            <a href="https://raven-os.org/">Go back to main site</a>
```

Comment:
You're using a direct link here. This is an issue because in a staging environment, the home page wouldn't be `raven-os.org` but maybe an other domain name, like `dev.raven-os.org`, `staging.raven-os.org` or whatever.

You should put a link to `/` instead, refering to the main page of the current website.

```html
<a href="/">Go back to home page</a>
```

(I tend to think that "home page" is better than "main site", but that is a matter of opinion).